docs/usage.md

```diff
@@ -3,10 +3,88 @@
+    - CVEs:
+        - CVE-2022-0847, CVE-2021-4034, CVE-2017-7269, CVE-2016-8562, CVE-2010-2568.
+    - ICS Advisory:
+        - USB Malware Targeting Siemens Control Software (Update C) - (ICSA-10-201-01C)
+    - On Creation of KEV ICS Advisory KEV Alert get created:
+        - ICSA-10-201-01C - Siemens
+    - Mandatory Tasks(Under Evaluation Tab):
+        - CVE-2010-2568 - State Of Exploitation **Urgent**
+        - CVE-2010-2568 - Technical Impact **Urgent**
+        - CVE-2010-2568 - Automatable **Urgent**
+        - CVE-2010-2568 - Mission Prevalence **Urgent**
+        - CVE-2010-2568 - Public Well-Being Impact **Urgent**
+        - CVE-2010-2568 - Mitigation Status **Urgent**
+        > **Note**: Mandatory task get created only for each KEV CVEs found in ICS Advisory.
+    - Recommended Tasks(Under Remediation Tab):
+        - ICSA-10-201-01C - Siemens - Make Permanent Configuration Changes
+        - ICSA-10-201-01C - Siemens - Isolating vulnerable systems, applications, services, profiles, or other assets
+        - ICSA-10-201-01C - Siemens - Limiting access
+- To get the latest report of CVEs from NIST-NVD execute playbook **Get Latest Report Of CVEs**
+- Identification tab contains the steps to be performed.
+- Execute the appropriate playbook that is available by task type to completed all the **Mandatory Tasks**.
+- After completing all **Mandatory Tasks**, execute **Calculate SSVC Score** to determine the score for each KEV CVE. Evaluation tab contains SSVC Score.
+![SSVC Score](/docs/res/ssvc_score.png)
+- Remediation tab also contains patch existence query as **Yes** or **No**. For each option related task get created.
+> **Note**: If query value get changed, related task for patch get deleted and new task get created. On selecting **unknown** related patch task get deleted.
+- Reporting & Notification tab contains the details of report and Additional Notes to be added in report.
+- To create a report, run **Generate Status Summary Report**.
+
+> **Note**: Please mark **Do Patch Exist?** as **unknown** before reseting scenario
+
+## Real Mode
+
+- Set global variable **Demo_mode** as **false**.
+- In **02 - Use Case - ICS Advisory and CVEs** and **02 - Use Case - KEV Alert**, mark all of the playbooks that are available as active.
+- Ingest KEV CVE using any of the below ways:
+    - Trigger the schedule **Ingest KEV CVEs** to get all the KEV CVEs from NIST-NVD.
+    - Execute **Ingest KEV CVEs** from CVE List View.
+    > **Note**: Please wait till ingestion gets over.
+- Playbook **Ingest KEV CVEs** contains **Configuration** step where you can modify variable **pageLimit** and **pageIndex** to acquire given number of data per page.
+- Ingest ICS Advisory using any of the below ways:
+    - Trigger the schedule **Ingest ICS Advisory** to get all ICS Advisroy 
+    - Execute **Ingest ICS Advisory** from ICS Advisory List View.
+    - Execute **Inject Advisory Through CSV File** from ICS Advisory List View.

```

Comment:
Add a sample CSV format for the reference

Reply:
added sample csv file